tests/console/firewalld.pm

```diff
@@ -340,6 +340,27 @@ sub test_firewall_offline_cmd {
     }
 }
 
+# Test #10 - Verify default backend on 15+ (SLES and Leap)
+#            Factory derived products, should have nf_tables too
+sub test_default_backend {
+    validate_script_output('iptables --version', sub {
+            # This could have been done using capture groups too
+            # removing the need for repeating regexes and nesting ifs
+            # let's wait until a new backend is added, before optimizing
+            if (uses_iptables) {
+                # if iptables reports no backend, or legacy backend, we' re using old version of it
+                m/(?:iptables\sv[[:digit:]].+\w)\s?($|(?:.legacy.$))/;
+            } else {
+                if (m/(?:iptables\sv[[:digit:]].+\w)\s?($|(?:.legacy.$))/) {
+                    record_soft_failure('bsc#1206383');
```

Comment:
```suggestion
                    record_soft_failure('bsc#1206383 - iptables uses legacy backend instead of nftables');
```
That would avoid looking up what the bsc is about. WDYT?